lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
@nzakas I mentioned those rules as potentially triggering `max-len` since they can introduce a character in a line which has already `max-len` chars in it, and it will also be hidden.